### PR TITLE
Add a bio file for @githubteacher

### PR DESCRIPTION
**Pull request by githubteacher**
This Pull Request adds a bio file for @githubteacher as requested in #2.

Hey, @githubstudent, I would :heart: if you had some :watch: to put some :eyes: on this and let me know your :thought_balloon:s.

### CODE REVIEW

**Comment by githubteacher** [comment]
These changes look good overall, but there are a few things that you might want to address.

File: docs/githubteacher.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# GitHub Teacher Bio
+
+- Name: GitHub Teacher
+- Email: trainingdemos+githubteacher@github.com
+- Occupation: Teacher
+- Location: San Francisco
```
Comment (githubteacher) [inline docs/githubteacher.md]: There are lots of San Franciscos in the :earth_africa:

**Comment by klsweet** [comment]
This looks wrong. You should go back to coding school. :-1: You need help. Maybe it is time to retire from coding all together. :+1: Start looking for a new job. :eyes:

**Comment by klsweet** [comment]
OMG, I just love all those emoticons, especially the :smirk: and the :rage: This is too much fun. I might not get any work done. :facepunch:

**Comment by klsweet** [comment]
You are great teacher. Thanks for doing such a great job with the GitHub training today. :+1: :smile:

**Comment by githubteacher** [comment]
:+1: 

:ship: it!

:shipit:

**Pull request merged**